test/OpenSTUtility.js

```diff
@@ -425,7 +425,7 @@ contract('OpenSTUtility', function(accounts) {
 			})
 
 			it('fails to process if msg.sender != redeemer', async () => {
-	            await Utils.expectThrow(openSTUtility.processRedeeming(redemptionIntentHash, { from: accounts[1] }));
+	            await Utils.expectThrow(openSTUtility.processRedeeming(redemptionIntentHash, { from: accounts[5] }));
```

Comment:
The `it` of this test should be updated to reflect what the comment implies—this tests that a non-redeemer/non-registrar cannot successfully call the function.

Reply:
changed to "fails to process if msg.sender is not redeemer or registrar"